### DISCUSSION TITLE
How to start numbering other than 1 in pdf? (tried number-offset: 2, but no success)

### DISCUSSION THREAD

**Discussion by entelecheia1963** [quarto-dev/quarto-cli#7701] [Q&A] [answered]
### Description

I am generating different chapters of a workbook using different .ipynb files. I want to sections numbers to start at a defined value rather than the default '1'. I have tried the below .. but that does work for me. What am I doing wrong?

````qmd
---
title : Heading numbering
subtitle : Want to start at level 2
author : Me
date: 25/11/2023
format: 
  pdf:
    toc: true
    number-offset: 2
    number-sections: true
    colorlinks: true
    papersize: A4
    lof: true
    code-line-numbers: true
highlight-style: zenburn
execute:
  echo: true
  code-overflow: wrap
---
````

````{markdown}
# Heading should be 2
## Should be 2.1
## should be 2.2
# should be 3
## should be 3.1

````

The result I get is this:

![image](https://github.com/quarto-dev/quarto-cli/assets/42785164/2c429e1f-a278-413a-9a74-00dd05a93dc9)

**Comment by mcanouil** [comment, marked as answer]
It seems to have an error in the reference pages.
`number-offset` is a Pandoc option which is only valid for HTML (<https://pandoc.org/MANUAL.html#options>) as it is no used in the template, thus the same applies to Quarto.

Now the question is, should a PR be made to update the LaTeX template on Pandoc side to be compatible with this option (cc @cderv).

Meanwhile you can use $\LaTeX$ to get offset.

```yml
include-in-header:
  - text: |
      \setcounter{section}{3}
```

**Reply by mcanouil** [reply]
~~I opened an issue to track this:~~
- ~~#7702~~

Actually, there was already one:
- #6842

**Reply by mcanouil** [reply]
I made a PR to update the documentation until there is really a support for this.
- #7703

**Comment by entelecheia1963** [comment]
That worked! Excellent! Thank you.
